scala-runner

```diff
@@ -30,6 +30,12 @@ getScalaNightly() {
   . <(curl https://raw.githubusercontent.com/scala/community-builds/"$1"/nightly.properties 2>/dev/null)
   echo "$nightly"
 }
+setResolver() {
+  case "$scala_version" in
```

Comment:
Could you make this the parameter (`$1`)?  Just seems too needy to depend on the global state.